receiver/gitproviderreceiver/internal/scraper/githubscraper/helpers.go

```diff
@@ -306,3 +308,37 @@ func (ghs *githubScraper) getCommitInfo(
 func getAge(start time.Time, end time.Time) int64 {
 	return int64(end.Sub(start).Seconds())
 }
+
+func (ghs *githubScraper) getCVEs(
+	ctx context.Context,
+	client graphql.Client,
+	repo string,
+) (*getRepoCVEsResponse, error) {
+	cves, err := getRepoCVEs(ctx, client, ghs.cfg.GitHubOrg, repo)
+	if err != nil {
+		ghs.logger.Sugar().Errorf("error getting CVE data from repo: %v", zap.Error(err))
```

Comment:
Maybe we should just return cve's & error since we do an error log at the calling function?

Reply:
My question also relates to needing this error message logged here.

Reply:
We have this error check with a same log message 1 layer up. It's duplication. I think we could just not check the error here and simply let the log and error check happen at the layer up. Thoughts?

Reply:
I see, you're right, there's a duplicate error message now. I can remove the log message here since the error still surfaces up in the return

Reply:
The error logging is done 1 layer up for the other functions so I think I'll keep it consistent with the others. No benefits to keeping it where it is

Reply:
Pushed the changes. Marking as resolved